TOR/PR_checklist.md

```diff
@@ -0,0 +1,17 @@
+# TC checklist for Pull Requests
+
+## Must criteria
+- [] CLA signed
+- [] Referenced EIP or issue exists
+- [] Description exists
+- [] PR addresses only one concern
+- [] PR can be built
+- [] Code formatting checks pass
+- [] Tests pass
+- [] No commented out code sections
+- [] English language used in code and comments
+- [] Code is understandable/maintainable
+
+## Optional criteria
+- [] Tests implemented
```

Comment:
How can tests pass if they are not implemented??

Reply:
Already implemented tests can pass. And they can fail if a bugfix or a enhancement has side effects.